apps/dashboard/src/app/(dashboard)/(chain)/chainlist/components/client/filters.tsx

```diff
@@ -29,7 +29,7 @@ export function AllFilters() {
       <PopoverTrigger asChild>
         <Button
           variant="outline"
-          className="h-10 w-auto p-2 gap-2 lg:px-4 lg:py-2 lg:h-10 lg:border-0"
+          className="w-auto h-10 gap-2 p-2 lg:px-4 lg:py-2 lg:h-10 lg:border-0"
```

Comment:
what is sorting these? Don't get me wrong its nice, but we should all have it so these diffs don't end up ruining every code review. :)